ui/mirage/factories/token.js

```diff
@@ -38,8 +38,19 @@ namespace "default" {
   policy = "read"
   capabilities = ["list-jobs", "alloc-exec", "read-logs"]
   secure_variables {
+    # full access to secrets in all project paths
+    path "project/*" {
+      capabilities = ["write", "read", "destroy", "list"]
+    }
+
+    # full access to secrets in all project paths
     path "*" {
-      capabilities = ["list"]
+      capabilities = ["write", "read", "destroy", "list"]
```

Comment:
thought: maybe we should leave the `*` capabilities for our demo token to just `read` and `list`? That way testers can experience what a limited experience looks like (and anything under `project/*` could be used to demonstrate higher-capability stuff like editing)